ota-plus-web/app/reactapp/src/components/dashboard/new/DashboardStepper/index.jsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import { useTranslation, Trans } from 'react-i18next';
+import { useObserver } from 'mobx-react';
+import { useStores } from '../../../../stores/hooks';
+import DashboardStepperCard from './DashboardStepperCard';
+import { CardsWrapper, Step, StepperWrapper, Stepper } from './styled';
+import { ICON_PATHS } from '../../../../config';
+import { ANT_STEP_STATUS, STEP_STATUS, STEP_TYPES } from '../../../../constants';
+import {
+  URL_STEPPER_CAMPAIGNS_CTA,
+  URL_STEPPER_CAMPAIGNS_DOCS,
+  URL_STEPPER_DEVICES_CTA,
+  URL_STEPPER_DEVICES_DOCS,
+  URL_STEPPER_GROUPS_CTA,
+  URL_STEPPER_GROUPS_DOCS,
+  URL_STEPPER_SOFTWARE_CTA,
+  URL_STEPPER_SOFTWARE_DOCS,
+  URL_STEPPER_UPDATES_CTA,
+  URL_STEPPER_UPDATES_DOCS
+} from '../../../../constants/urlConstants';
+
+function useStoreData() {
+  const { stores } = useStores();
+  return useObserver(() => ({
+    campaignsTotal: stores.campaignsStore.campaignsTotal,
+    campaignsWithErrorsTotal: stores.campaignsStore.campaignsWithErrorsTotal,
+    devicesTotalCount: stores.devicesStore.devicesTotalCount,
+    notSeenRecentlyDevicesTotal: stores.devicesStore.notSeenRecentlyDevicesTotal,
+    ungroupedDevicesTotal: stores.devicesStore.ungroupedDevicesTotal,
+    groupsTotal: stores.groupsStore.groupsTotal,
+    versionsTotal: stores.softwareStore.versionsTotal,
+    updatesTotalCount: stores.updatesStore.updatesTotalCount,
+  }));
+}
+
+const DashboardStepper = () => {
+  const { t } = useTranslation();
+  const {
+    devicesTotalCount,
+    notSeenRecentlyDevicesTotal,
+    ungroupedDevicesTotal,
+    versionsTotal,
+    groupsTotal,
+    updatesTotalCount,
+    campaignsTotal,
+    campaignsWithErrorsTotal
+  } = useStoreData();
+
+  const LINKS_DATA = [
+    {
+      cta: URL_STEPPER_DEVICES_CTA,
```

Comment:
What do you mean by shortcut cta?

Reply:
call to action, it's in reference to a cta button visible on each card